libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -62,18 +69,42 @@ export class FixedCardLayoutComponent implements AfterViewInit {
     private numberOfColumns = 3;
     /** @hidden */
     private previousNumberOfColumns: number;
+    private differ: any;
 
     /** @hidden */
-    @HostListener('window:resize', [])
     onResize(): void {
         this.numberOfColumns = this.getNumberOfColumns();
         if (this.previousNumberOfColumns !== this.numberOfColumns) {
-            this.previousNumberOfColumns = this.numberOfColumns;
-            this._renderLayout();
+            this.ngZone.run(() => {
+                this.previousNumberOfColumns = this.numberOfColumns;
+                this._renderLayout();
+            });
         }
     }
 
-    constructor(private _cd: ChangeDetectorRef, public elementRef: ElementRef) {}
+    constructor(
+        private readonly _cd: ChangeDetectorRef,
+        private readonly elementRef: ElementRef,
+        private readonly ngZone: NgZone,
+        differs: IterableDiffers
+    ) {
+        this.differ = differs.find([]).create(null);
+    }
+
+    /** @hidden */
+    ngOnInit(): void {
+        this.ngZone.runOutsideAngular(() => {
+            this.elementRef.nativeElement.addEventListener('window:resize', this.onResize.bind(this));
```

Comment:
It does not work.
There is no `window:resize` event on HTMLElement.

Reply:
fixed